screwdriver/scripts/publish.sh

```diff
@@ -38,5 +38,8 @@ deployProject "com.yahoo.athenz:athenz-zpe-java-client"
 deployProject "com.yahoo.athenz:athenz-msd-java-client"
 deployProject "com.yahoo.athenz:athenz-server-common"
 deployProject "com.yahoo.athenz:athenz-instance-provider"
+deployProject "com.yahoo.athenz:athenz-dynamodb-client-factory"
```

Comment:
not sure if we want to deploy these three packages yet. they're not client functionality that other teams need to use. the team that wants to build and deploy a syncer can build, package and deploy without using maven central.

Reply:
fixed